include/umf/memory_pool.h

```diff
@@ -37,6 +37,7 @@ typedef enum umf_pool_create_flag_t {
          << 0), ///< Pool will own the specified provider and destroy it in umfPoolDestroy
     UMF_POOL_CREATE_FLAG_DISABLE_TRACKING =
         (1 << 1), ///< Pool will not track memory allocations
+    UMF_POOL_CREATE_FLAG_PROXY_POOL = (1 << 2), ///< Reserved for UMF Proxy Pool
```

Comment:
Why do we need proxy lib specific flag in underlying libumf? I do not think it is good design decision - it might confuse users of libumf who does not know about proxy lib.

I think we need to discuss this at our umf technical meeting we need to define the design and agree on that before changing the interface.

Reply:
changed - now I define a "reserved" flag for this case

Reply:
What does it mean for other users of `libumf`?
So I think we need to discuss the design of the proxy lib and interaction with core libumf at the UMF technical meeting.

Reply:
sure we can